Hunter/Assets/Scripts/Model/Entities/Herd.cs

```diff
@@ -16,12 +16,11 @@ public Herd()
             {
                 _animals[i] = new Deer();
             }
-
         }
 
-        public static List<Animal> CreateAnimals(int numberOfHerds)
+        public static List<Entity> CreateEntities(int numberOfHerds)
         {
-            var herds = new List<Animal>();
+            var herds = new List<Entity>();
             for (var i = 0; i < numberOfHerds; i++)
```

Comment:
Thirdly - very big change. Now CreateAnimals in both Herd and Rabbit creates **Entities** instead of **Animals**, 'cause Our game has also a **Hunter** who's not an animal